docs/security.rst

```diff
@@ -58,30 +58,183 @@ SSL
 ---
 SSL should be used when client encryption is enabled in Cassandra.
 
-Versions 3.17.0 and higher
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-To enable SSL you will need to set :attr:`.Cluster.ssl_context` to a
-``ssl.SSLContext`` instance. Optionally, you can also set :attr:`.Cluster.ssl_options`
+With version 3.17.0 and higher, you will need to set :attr:`.Cluster.ssl_context` to a
+``ssl.SSLContext`` instance to enable SSL. Optionally, you can also set :attr:`.Cluster.ssl_options`
 to a dict of options. These will be passed as kwargs to ``ssl.SSLContext.wrap_socket()``
 when new sockets are created.
```

Comment:
Just a nit, I feel like we can describe this API more positively. "To give you as much control as possible over your SSL configuration, our SSL API takes a user-created `SSLContext` object from the Python standard library. These docs will include some examples for how to achieve common configurations, but the `ssl.SSLContext` documentation gives a more complete description of what is possible."

Reply:
I like that.